### PR TITLE
faq: Add help to configure Tigase for providing properties via XMPP

### PR DESCRIPTION
**Pull request by melvo**

### CODE REVIEW

File: content/faq.md [hunk 1]
```diff
@@ -111,6 +112,7 @@ For retrieving a web page that can be used to create accounts, [XEP-0077: In-Ban
 Please use that feature to either provide an alternative way for users to create an account or if your server does not support account creation via XMPP apps:
 * ejabberd: [mod_register](https://docs.ejabberd.im/admin/configuration/modules/#mod-register) - `redirect_url`
 * Prosody: [mod_register_oob_url](https://modules.prosody.im/mod_register_oob_url.html) - `register_oob_url`
+* Tigase: *Unsupported*
```
[the remark this answers is not in the document]
Comment (melvo) [reply content/faq.md:115]: Where is that documented? Where can the URL be set or the web registration enabled?

File: content/faq.md [hunk 2]
```diff
@@ -127,6 +130,7 @@ For retrieving the support addresses of a provider, [XEP-0157: Contact Addresses
 Please provide those addresses to enable users and our upcoming support bots to reach you (**Important: Append `?join` to group chat addresses**):
 * ejabberd: [mod_disco](https://docs.ejabberd.im/admin/configuration/modules/#mod-disco) - `server_info`, entry with `name: support-addresses`
 * Prosody: [mod_server_contact_info](https://prosody.im/doc/modules/mod_server_contact_info) - `support`
+* Tigase: [disco-extensions](https://docs.tigase.net/en/latest/Tigase_Administration/Configuration/_Configuration.html#abuse-contacts) - `support-addresses`
```
[the remark this answers is not in the document]
Comment (melvo) [reply content/faq.md:133]: Is it `support-address` or `support-addresses`? We only use the support addresses. Thus, I mentioned only them :) BTW: Maybe you could adjust your documentation to be more generic and speak about contact addresses while providing abuse addresses as an example.

File: content/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Echolon) [reply content/faq.md:115]: @woj-tek have you had time to review?

File: content/faq.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (melvo) [reply content/faq.md:133]: Alright! I could not see a change in the documentation. I suppose it must be deployed first. Will the URL (https://docs.tigase.net/en/latest/Tigase_Administration/Configuration/_Configuration.html#abuse-contacts) stay the same?

File: content/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (melvo) [reply content/faq.md]: Thanks for clarification! I updated the entry.

**Pull request merged**